### ISSUE TITLE
wunjo.org is parked

### ISSUE THREAD

**Issue by zmt** [jcorbin/zsh-git#13]
The http://www.wunjo.org/zsh-git referenced in the README is now 404.

**Comment by jcorbin** [comment]
yes, I am bad webmaster ; should move to github pages, thanks for the reminder!
